CefSharp.Test/BindingRedirectAssemblyResolver.cs

```diff
@@ -28,13 +27,25 @@ void IDisposable.Dispose()
         private static Assembly CurrentDomainAssemblyResolve(object sender, ResolveEventArgs args)
         {
             var asemblyName = new AssemblyName(args.Name);
+
+            // during tests and test discovery outside of visual studio the Environment.CurrentDirectory points to the output directory
             var path = Path.Combine(Environment.CurrentDirectory, asemblyName.Name + ".dll");
 
             if (File.Exists(path))
             {
                 return Assembly.LoadFrom(path);
             }
 
+            // during test discovery inside of visual studio the current directory points to e.g. "C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\Common7\IDE"
+            // which is why we are also checking the executing assembly which points to the output directory
+            // strangely enough though the execution assembly is different when test discovery happens outside of visual studio but in that case the Environment.CurrentDirectory is correct...
+            path = Path.Combine(Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location), asemblyName.Name + ".dll");
```

Comment:
Can we use the location of the CefSharp.Test assembly as a base to handle both cases?

Reply:
Sadly no, because when Visual Studio discovers test in the background it is copies the dll to a temp location without the dependencies. See the first table:

typeof(BindingRedirectAssemblyResolver).Assembly.Location | C:\Users\xxx\AppData\Local\Temp\797da606-0fcc-492f-83ae-9649848339e7\797da606-0fcc-492f-83ae-9649848339e7\assembly\dl3\c971822d\79d3fec4_95f0d501\CefSharp.Test.dll
-- | --

Reply:
But we can use `typeof(BindingRedirectAssemblyResolver).Assembly.Location` instead of `Assembly.GetExecutingAssembly().Location` for this specific case.